modules/browser/BrowserDetection.js

```diff
@@ -0,0 +1,305 @@
+import bowser from 'bowser';
+import { getLogger } from 'jitsi-meet-logger';
+
+import {
+    CHROME,
+    OPERA,
+    FIREFOX,
+    INTERNET_EXPLORER,
+    EDGE,
+    SAFARI,
+    NWJS,
+    ELECTRON,
+    REACT_NATIVE
+} from './browsers';
+
+const logger = getLogger(__filename);
+
+/**
+ * Maps the names of the browsers from bowser to the internal names defined in
+ * ./browsers.js
+ */
+const bowserName2JitsiName = {
```

Comment:
Could we call this bowserNameToJitsiName instead? "to" and "2" are not the same to me.

Reply:
Sure!